tests/tests_integration/test_api/test_relationships.py

```diff
@@ -17,16 +17,19 @@
 
 
 @pytest.fixture
-def new_relationship():
+def new_relationship(new_label):
     external_id = uuid.uuid4().hex[0:20]
+    pre_existing_data_set = 3603480159997264
```

Comment:
i guess this means the tests will fail if run on any other tenant than the sdk integration test tenant?

Reply:
yes. I couldn't think of any other way to by pass the mess of data-sets 😅

Reply:
its good to test that data sets are working, but i would argue for either creating the data set with an external id, and then getting the internal id by fetching the data set, or setting the data set id by an env variable, although neither solution is very clean - how are data set tests handled elsewhere in the sdk?

Reply:
There is no test that includes data sets other than data set integration tests. Nevertheless I agree it should be fetched. So changed that part.